Mathlib/Data/Sym/Sym2.lean

```diff
@@ -764,4 +764,8 @@ instance [IsEmpty α] : IsEmpty (Sym2 α) :=
 instance [Nontrivial α] : Nontrivial (Sym2 α) :=
   diag_injective.nontrivial
 
+-- TODO: use a sort order if available, gh-18166
```

Comment:
Let's use full URLs when referencing issues or prs.

Actually, what is this, the number is too high to be an issue??

Reply:
```suggestion
-- TODO: use a sort order if available, https://github.com/leanprover-community/mathlib/issues/18166
```